Kitodo/src/main/java/org/kitodo/services/data/TemplateService.java

```diff
@@ -251,6 +260,11 @@ public void addAllObjectsToIndex() throws CustomResponseException, InterruptedEx
         indexer.performMultipleRequests(findAll(), templateType);
     }
 
+    @Override
+    public TemplateDTO convertJSONObjectToDTO(JSONObject jsonObject, boolean related) throws DataException {
+        return new TemplateDTO();
```

Comment:
Please add a TODO annotation that mentioned that this method is not full implemented.